stats/bin/spark-stats.js

```diff
@@ -11,6 +12,12 @@ const {
 } = process.env
 
 const pgPools = await getPgPools()
+
+// Refresh the leaderboard's materialized view every 12 hours
+setInterval(async () => {
+  await updateTopMeasurementStations(pgPools.evaluate)
+}, 1000 * 60 * 60 * 12)
```

Comment:
IIRC, spark-stats has read-only access to the DB; it's connected to a reader replica to reduce the load on the main DB server instance.

Reply:
Moved it to spark-evaluate PR!